### PR TITLE
change home->base throughout docs

### PR DESCRIPTION
**Pull request by tinnus-napbus**
this updates all reference to the home desk except for the ford explanation and some paths printed in the jetting doc

regarding #1304

### CODE REVIEW

File: content/docs/hoon/guides/cli-tutorial.md [hunk 1]
```diff
@@ -264,14 +272,15 @@ a type or list of types that contain data needed to execute the command. See
 `card` is either an ordinary Gall agent `card` or a `%shoe` `card`, which takes
 the shape `[%shoe sole-ids=(list @ta) effect=shoe-effect]`. A `%shoe` `card` is
 sent to all `sole`s listed in `sole-ids`, imaking them run the `sole-effect`
-specified by `effect` (i.e. printing some text).  Here we can
-reference `card:shoe` because of `/+  shoe` at the beginning of the app.
+specified by `effect` (i.e. printing some text). Here we can
+reference `card:shoe` because of `/+ shoe` at the beginning of the app.
```
Comment (drbeefsupreme) [inline content/docs/hoon/guides/cli-tutorial.md]:
```suggestion
reference `card:shoe` because of `/+  shoe` at the beginning of the app.
```

**Review by drbeefsupreme — CHANGES_REQUESTED** [review]

**Review by drbeefsupreme — APPROVED** [review]

**Comment by drbeefsupreme** [comment]
I do think the Ford post should be altered as I wrote in #1304 but I'm not adamant enough about it to stop this from being merged.

**Comment by drbeefsupreme** [comment]
Not sure why the build is failing though.

**Comment by tinnus-napbus** [comment]
hmm I merged master and now it builds, dunno what's up with that. Also sorry about of the weird whitespace changes, it seems markdown-mode does it automatically. I would note, though, that whitespace is collapsed for inline code, so the space you've re-added will not be displayed

**Pull request merged**